tests/routes/native/v1/offerers_test.py

```diff
@@ -25,6 +26,7 @@ def test_get_venue(self, client):
             "withdrawalDetails": venue.withdrawalDetails,
             "address": venue.address,
             "postalCode": venue.postalCode,
+            "venueTypeEnum": venue.venueType.code,
```

Comment:
```suggestion
            "venueType": venue.venueType.code,
```

+ mettre une enum plutôt qu'un string comme expliqué au dessus